assets/enx-redirect/report/index.html

```diff
@@ -53,7 +59,9 @@ <h1>Request Exposure Notifications verification code from {{.realm.Name}} ({{.re
                  <label for="agreement" class="form-check-label">
                    I agree that I have received a positive test result for COVID-19, that the phone
                    number provided belongs to me, and that I would like to receive a verification code
-                   for Exposure Notifications to share my information anonymously with others. 
+                   for Exposure Notifications to share my information anonymously with others. I understand
+                   that my phone number may be saved temporarily for anti-abuse purposes and this phone number
```

Comment:
I understand that a one-way hash of my phone number... ?

Reply:
I don't think it will confuse the average user, but I think it will be used against us in the press.